models.yml

```diff
@@ -1011,6 +1011,10 @@ meeting:
     type: boolean
     default: False
     restriction_mode: B
+  list_of_speakers_hide_secondary_contributions_count:
```

Comment:
Mh, I think something like `hide_contribution_numbering` or `hide_contribution_count` would fit better - I don't think "secondary" is a good descriptor for all contributions after the first...